src/test/java/Homework18CH.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18CH extends BaseTest{
+
+    @Test
+    public void playSongInHomepage() throws InterruptedException {
+        login();
+        Thread.sleep(2000);
+        homePage();
+        Thread.sleep(2000);
+        mostPlayed();
+        Thread.sleep(3000);
+        playSong();
+        Thread.sleep(2000);
+        songIsPlaying();
```

Comment:
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.